README.md

```diff
@@ -319,6 +319,51 @@ warnings = validate.validate_pantry(pantry)
 pantry.validate()
 ```
 
+### Using A Mongo DB
+
+There are a couple of things to add when using mongo with weave. 
+You need to give a Pantry object a mongo client.
+
+```python
+# gets a connection to the mongo db
+client = weave.config.get_mongo_db()
+# giving the Pantry object the mongo client
+demo_pantry = weave.Pantry(
+    weave.IndexPandas,
+    pantry_path="mongodb-demo",
+    file_system=s3fs.S3FileSystem(client_kwargs = {'endpoint_url': os.environ['S3_ENDPOINT']}),
+    mongo_client=client,
+)
+```
+
+It is the same process as before expect adding teh mongo clinet to the Pantry object.
```

Comment:
"the" is spelled wrong.

I think you can just omit "It is the same process as before expect adding teh mongo clinet to the Pantry object."